nmproxy/router/firewall_nonlinux.go

```diff
@@ -40,6 +40,17 @@ func (unimplementedFirewall) FlushAll() {
 
 }
 
+func (unimplementedFirewall) InsertEgressRoutingRules(server string, egressInfo models.EgressInfo) error {
```

Comment:
missing comments  on functions

Reply:
it's unimplemented firewall funcs, so comments on these funcs are not required